.github/workflows/php-compatibility.yml

```diff
@@ -0,0 +1,55 @@
+name: PHP Compatibility
+
+env:
+  COMPOSER_VERSION: "2"
+  COMPOSER_CACHE: "${{ github.workspace }}/.composer-cache"
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  php_compatibility:
+    name: ${{ matrix.php }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [ '5.6', '7.0', '7.4', '8.0', '8.1' ]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set standard 10up cache directories
+        run: |
+          composer config -g cache-dir "${{ env.COMPOSER_CACHE }}"
+
+      - name: Prepare composer cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.COMPOSER_CACHE }}
+          key: composer-${{ env.COMPOSER_VERSION }}-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            composer-${{ env.COMPOSER_VERSION }}-
+
+      - name: Set PHP version
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.2'
+          coverage: none
+          tools: prestissimo, composer:v2
```

Comment:
`prestissimo` is unnecessary if we're using composer v2,
[Refer to this](https://github.com/hirak/prestissimo#announcement-composer-2-is-now-available) for more details.